state/backups/backups_linux.go

```diff
@@ -50,6 +50,7 @@ func ensureMongoService(agentConfig agent.Config) error {
 		numaCtlPolicy,
 		agentConfig.MongoVersion(),
 		true,
+		mongo.MemoryProfileDefault,
```

Comment:
the service is going to be re-written once the restore operation is over, for this particular operation I prefer to unleash mongo ram consumption.